samples/Extensions/CosmosDB/CosmosDbInputBindingFunction.cs

```diff
@@ -24,6 +24,7 @@ public CosmosDbInputBindingFunction(ILogger<CosmosDbInputBindingFunction> logger
         }
 
         [Function("DocByIdFromJSON")]
+        [FixedDelayRetry(5, "00:00:10")]
```

Comment:
I believe retry attributes are only meant for cosmosdb triggers - in this case while the input is cosmosdb, the trigger is a queue trigger.

Reply:
fixed!